pycape/cape.py

```diff
@@ -111,6 +112,41 @@ def connect(self, function_ref, function_token=None):
         function_ref = _convert_to_function_ref(function_ref)
         self._loop.run_until_complete(self._connect(function_ref, function_token))
 
+    @contextlib.contextmanager
+    def connection_context(self, function_ref, function_token=None):
```

Comment:
toying with the idea of calling this `function_context`, since a connection is 1:1 with a function. considering the only thing that the context is used for is to be able to repeatedly call that particular function. wdyt?

Reply:
good point works for me